packages/babel-plugin-transform-es3-reserved-words/src/index.js

```diff
@@ -0,0 +1,16 @@
+import { es3, es2017 } from "./reserved-words";
+
+const difference = (arr, arr2) => arr.filter(el => !(arr2.indexOf(el) !== -1));
+const reservedKeywords = difference(es3, es2017);
+
+export default function() {
+  return {
+    visitor: {
+      "BindingIdentifier|ReferencedIdentifier"(path) {
+        if (reservedKeywords.indexOf(path.node.name) !== -1) {
```

Comment:
Babel is targeting Node >= 4 now, so you can use `Set` with `.has` here instead of an array.